tests/test_horizon.py

```diff
@@ -0,0 +1,77 @@
+from io import StringIO
+import sys
+import unittest
+import numpy as np
+
+from prog_models.models.thrown_object import ThrownObject
+from prog_algs import *
+from pprint import pprint
```

Comment:
```suggestion
from io import StringIO
import sys
import unittest

from prog_models.models.thrown_object import ThrownObject
from prog_algs import *
```

Remove unused imports